source/includes/_permissions.md

```diff
@@ -326,6 +326,15 @@ Requires one of the following:
 - domain permission: `edit_dataset`
 - current dataset permission: `edit_dataset`
 
+### Dataset Resource Download
+
+`GET /api/management/v2/datasets/<dataset_uid>/resources/<resource_uid>/download`
+
+Requires one of the following:
+
+- domain permission: `edit_dataset`
+- current dataset permission: `edit_dataset` or `publish_dataset`
```

Comment:
The permission to publish the dataset doesn't allow the user to download a resource file

Reply:
OK, I'll remove the `publish_dataset` from this line.